package main.modelo.excepciones;

public class RespuestaVaciaException extends Exception {
	public RespuestaVaciaException() {
		super("Vaya, a√∫n no ha sido respondida la pregunta");
	}
}
